docs/KEYS.md

```diff
@@ -0,0 +1,15 @@
+# Trusted keys
```

Comment:
We should also add an instruction how to verify the release. Just a reference to some guide I guess would be enough. But this can be done later.

Reply:
I think we should modify `docs/installer/lokoctl.md` to reflect this. We'll do that later.